### DISCUSSION TITLE
How to (bundle and) deploy only the `server/api` without the rendering of pages

### DISCUSSION THREAD

**Discussion by posva** [nuxt/nuxt#19661] [Questions]
By default, the renderer (with vue) is also bundled into the server bundle. I prerender some pages and make the rest client only with:

```ts
  routeRules: {
    '/': { prerender: true },
    '/login': { ssr: false },
    // ... other routes
    '/api/**': { ssr: true },
  },
```

in the nuxt config. I'm deploying to vercel but I don't think that changes anything.

The reason I want to remove the renderer is because the folder is getting too big with server dependencies and removing the renderer and other code required to SSR would allow me to save about 5mb

**Comment by danielroe** [comment]
We discussed this on Monday as a team in relation to https://github.com/nuxt/nuxt/issues/14669#issuecomment-1465881507 and think it would be nice to offer this feature.
